block_iagora.php

```diff
@@ -55,9 +55,49 @@ public function get_content() {
         } else {
             $this->content->text = $this->generate_chat_content($copilotendpointurl);
         }
+
+          // Marquer l'activité comme vue pour la complétion si le suivi est activé
```

Comment:
```suggestion
        // Mark activity as viewed if tracking is enabled.
```
As most of our comments are in English, could we keep it that way?

Reply:
quite right